File: Classik_Tracks_Crawling/src/main/java/com/ssafy/a604/entity/Composer.java
package com.ssafy.a604.entity;

public class Composer {

    private final String name;
    private final String description;

    public Composer(String composer, String description) {
        this.name = composer;
        this.description = description;
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    @Override
    public String toString() {
        return "Composer [name=" + name + ", description=" + description + "]";
    }

}

File: Classik_Backend/src/main/kotlin/com/ssafy/Classik_Backend/entity/Composer.kt
package com.ssafy.Classik_Backend.entity

import jakarta.persistence.*

@Entity
class Composer(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "composer_id", nullable = false)
    val id: Int,

    val name: String,

    @Column(name = "image_url", nullable = false)
    val imageUrl: String,

    @Column(columnDefinition = "text")
    val description: String,

    @OneToMany(mappedBy = "composer", cascade = [CascadeType.ALL], orphanRemoval = true)
    val tracks: MutableSet<Track> = mutableSetOf(),
)